config/funding.js

```diff
@@ -14,50 +15,50 @@ fundingAmounts[chainAddressConstants.masterInternalFunderKind] = {
   originGas: {
     // origin deployer
     [chainAddressConstants.originDeployerKind]: {
-      fundAmount: '0.2941',
-      thresholdAmount: '0.1541'
+      fundAmount: coreConstants.FUND_FOR_CHAIN_SETUP ? '0.02810' : '0.01400',
+      thresholdAmount: coreConstants.FUND_FOR_CHAIN_SETUP ? '0.01405' : '0.0070'
     },
 
     // origin anchor owner
     [chainAddressConstants.originAnchorOrgContractOwnerKind]: {
-      fundAmount: '0.00006',
-      thresholdAmount: '0.00006'
+      fundAmount: coreConstants.FUND_FOR_CHAIN_SETUP ? '0.00006' : '0',
```

Comment:
why would this sometimes be zero?
when should `SA_FUND_FOR_CHAIN_SETUP` be false?